nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -47,23 +47,29 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         """Initializes the BERTTextClassifier model.
         """
 
-        self.max_seq_length = cfg.language_model.max_seq_length
-        self.data_dir = cfg.data_dir
         self.tokenizer = get_tokenizer(
             tokenizer_name=cfg.language_model.tokenizer,
             pretrained_model_name=cfg.language_model.pretrained_model_name,
             vocab_file=cfg.language_model.vocab_file,
             tokenizer_model=cfg.language_model.tokenizer_model,
-            do_lower_case=cfg.language_model.do_lower_case,
+            do_lower_case=cfg.dataset.do_lower_case,
         )
 
+        self.model_cfg = cfg
+        # shared params for dataset and data loaders
```

Comment:
we don't need this, after super.init() self._cfg == cfg

Reply:
removed.